packages/table/src/ExpandedContent/ExpandedContent.spec.tsx

```diff
@@ -91,3 +91,26 @@ describe('packages/table/Row/ExpandableContent', () => {
     expect(queryByText('Expandable content test')).toBeInTheDocument();
   });
 });
+
+describe('packages/table/Row/ExpandableContent/Disabled Animations', () => {
```

Comment:
this describe block should be nested inside `'packages/table/Row/ExpandableContent'`